jerry-core/ecma/base/ecma-helpers.c

```diff
@@ -576,26 +576,13 @@ ecma_get_internal_property (ecma_object_t *object_p, /**< object descriptor */
 ecma_property_t *
 ecma_create_named_data_property (ecma_object_t *object_p, /**< object */
                                  ecma_string_t *name_p, /**< property name */
-                                 bool is_writable, /**< 'Writable' attribute */
-                                 bool is_enumerable, /**< 'Enumerable' attribute */
-                                 bool is_configurable) /**< 'Configurable' attribute */
+                                 uint8_t prop_attributes) /**< property attributes */
```

Comment:
I just saw that we already have a `ecma_property_flags_t`, but it is an enum, so it'll be 32bit long (usually, but not always). In this case I think it is unnecessary to create a new type, but it should be mentioned in the comment.